examples/distributed/deepseek_deepep/intranode/dispatch.py

```
@@ -0,0 +1,972 @@
+
+@tilelang.jit(pass_configs={
+    "tl.disable_tma_lower": True,  # enable TMA later
+    "tl.disable_warp_specialized": True
+})
+def cached_dispatch_kernel(
+    num_ranks,
+    num_tokens,
+    num_max_send_tokens,  # config.num_max_nvl_chunked_send_tokens
+    num_recv_buffer_tokens,  # config.num_max_nvl_chunked_recv_tokens
+    hidden,
+    num_topk,
```

Ruff (0.14.8)

Unused function argument: `num_topk`

(ARG001)